.vib/spring-cloud-skipper/goss/spring-cloud-skipper.yaml

```
@@ -0,0 +1,14 @@
+command:
+  check-spring-cloud-skipper:
+    timeout: 20000
+    exec: timeout --preserve-status 15 /opt/bitnami/java/bin/java -jar /opt/bitnami/spring-cloud-skipper/spring-cloud-skipper.jar
+    exit-status: 143
+    stdout:
+      - Started
```

As this is the only application test we have for this asset, maybe we should use a narrower match:
```suggestion
      - "Started SkipperServerApplication"
```